### PR TITLE
Negative numbers are allowed in databindings.

### PR DESCRIPTION
**Pull request by rictic**
Fixes https://github.com/Polymer/polymer-linter/issues/66

 - [x] CHANGELOG.md has been updated

### CODE REVIEW

File: src/polymer/expression-scanner.ts [hunk 1]
```diff
@@ -164,7 +161,20 @@ export abstract class DatabindingExpression {
   }
 
   private _extractAndValidateSubExpression(
-      expression: estree.Node, callAllowed: boolean) {
+      expression: estree.Node, callAllowed: boolean): void {
+    if (expression.type === 'UnaryExpression') {
```
Comment (FredKSchott) [inline src/polymer/expression-scanner.ts]: combine conditionals

File: src/test/polymer/expression-scanner_test.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ suite('ExpressionScanner', () => {
             <template is="dom-if">
               <div id="[[bar]]"></div>
             </template>
-            <div id="{{bada(wing, daba.boom, 10)}}"></div>
+            <div id="{{bada(wing, daba.boom, 10, -20)}}"></div>
```
Comment (FredKSchott) [inline src/test/polymer/expression-scanner_test.ts]: "bada wing, daba boom!" -- Ancient Proverb

File: src/polymer/expression-scanner.ts (hunk 1 above)
Comment (rictic) [inline src/polymer/expression-scanner.ts]: +1

File: src/test/polymer/expression-scanner_test.ts (hunk 2 above)
Comment (rictic) [inline src/test/polymer/expression-scanner_test.ts]: https://www.youtube.com/watch?v=SEodHWBzDSU

**Pull request merged**